source/fundamentals/serialization/poco.txt

```diff
@@ -382,50 +382,86 @@ Specify an ID Generator
 
 Every document in a MongoDB collection must have a unique ID. When you serialize an object
 to a collection, if the ``Id`` property contains the default
-value for its data type, the {+driver-short+} doesn't serialize it. Instead, the driver
-generates a unique ID value for the property.
+value for its data type (usually ``null``), the {+driver-short+} doesn't serialize the
+default value. Instead, the driver generates a unique ID value for the property.
 
-If the ``Id`` property is of type ``Guid``, ``ObjectId``, or ``string``, the driver can
-generate the ID value on its own. If the ``Id`` property is any other data type, you must
-specify the ``IIdGenerator`` type that the driver uses to generate the value.
-To specify the ``IIdGenerator`` type, apply the ``[BsonId(IdGenerator)]`` attribute
-to the property and pass the ``IIdGenerator`` type as an argument.
-
-The {+driver-short+} includes the following ``IIdGenerator`` types:
+The following table describes the generators available in the {+driver-short+}:
 
 .. list-table::
    :header-rows: 1
-   :widths: 10 10
+   :widths: 25 75
 
    * - ``Id`` Field Data Type
```

Comment:
I can't immediately see why, but this is rendering without a space after "Id". Maybe because of the monospace?

Reply:
i couldn't get this to look right so i just removed the monospace. seems like that actually matches the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure) too